website/blog/2025-10-24-macos-xdg-cli-conventions.md

```
@@ -0,0 +1,179 @@
+
+1. Check your current credentials location:
+   ```bash
+   ls -la ~/Library/Application\ Support/atmos
+   ls -la ~/.config/atmos
+   ```
+
+2. Open an issue on [GitHub](https://github.com/cloudposse/atmos/issues) if you need help
+
+## References
+
+- XDG Base Directory Specification: https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)